src/modules/mavlink/streams/MANUAL_CONTROL.hpp

```diff
@@ -84,6 +84,36 @@ class MavlinkStreamManualControl : public MavlinkStream
 				msg.buttons |= (manual_control_switches.kill_switch << (shift * 6));
 			}
 
+			if (PX4_ISFINITE(manual_control_setpoint.aux1)) {
```

Comment:
Shouldn't we handle else case and prevent from using that aux channel?
In case of `manual_control_setpoint.aux2` being set to `NAN`